RNA_DNA_classes.py

```diff
@@ -0,0 +1,34 @@
+class Rna(str):
+    base_pairs = {'A': 'U', 'U': 'A',  'G': 'C', 'C': 'G', 'a': 'u', 'u': 'a',  'g': 'c', 'c': 'g'}
+
+    def __init__(self, sequence):
+        self.sequence = sequence
+        for base in self.sequence:
+            if base not in self.base_pairs:
+                raise ValueError('RNA sequence can contain only A, U, G and C bases')
+
+    def gc(self):
+        return (self.count('G')+self.count('g')+self.count('C')+self.count('c')) / len(self) * 100
+
+    def reverse_complement(self):
+        complement = ''
+        for base in self:
+            complement += self.base_pairs[base]
+        return complement[::-1]
+
+
+class Dna(Rna):
+    base_pairs = {'A': 'T', 'T': 'A', 'G': 'C', 'C': 'G', 'a': 't', 't': 'a', 'g': 'c', 'c': 'g'}
+
+    def __init__(self, sequence):
+        self.sequence = sequence
+        for base in self.sequence:
+            if base not in self.base_pairs:
+                raise ValueError('DNA sequence can contain only A, T, G and C bases')
+
+    def transcribe(self):
```

Comment:
Do we treat DNA as sense or antisense? If the second one, why we do not reverse the sequence?

Reply:
Usually we expect that DNA would be treated as sense strand, so changing 'T' to 'U' would be enough.
But if it is antisense (as I treated it), yes, reversing is necessary as nucleic acids should be written in 5'-3' way.